kaa/engine.pxi

```diff
@@ -1,13 +1,26 @@
+from libcpp.vector cimport vector
+
 from .kaacore.scenes cimport CScene
-from .kaacore.types cimport CRectangle
+from .kaacore.display cimport CDisplay
 from .kaacore.engine cimport CEngine, get_c_engine
 
 cdef Engine engine = None
 
 
 def get_engine():
     global engine
-    assert engine is not None
+
+    if engine is not None:
```

Comment:
in case engine gets destroyed inside kaacore this will return faulty wrapper which would segfault upon usage.